packages/cli/src/commands/info/info.ts

```diff
@@ -1,21 +1,14 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @flow
- */
-
+// @ts-ignore untyped
 import envinfo from 'envinfo';
 import {logger} from '@react-native-community/cli-tools';
-import type {ConfigT} from 'types';
+import {Config} from '@react-native-community/cli-types';
+// @ts-ignore - JS file
 import releaseChecker from '../../tools/releaseChecker';
 
 const info = async function getInfo(
-  argv: Array<string>,
-  ctx: ConfigT,
-  options: {},
+  _argv: Array<string>,
+  ctx: Config,
+  _options: {},
```

Comment:
```suggestion
```
If it's unused, we can remove it :)

Reply:
👍 wasn't sure if we preferred the same API for all commands but I see now a lot of the commands skip it